src/content/en/android/trusted-web-activity/quick-start/index.md

```diff
@@ -187,6 +174,9 @@ key:
        section.
     5. Use this value in your Digital Asset Link file.
 
+It is possible to have both keys in the `assetlinks.json`. See
+[Adding More Keys](#adding-more-keys) below for more information on how to do it.
```

Comment:
So it turns out I made a mistake in the section you linked here (you can't comment on lines that weren't changed on GitHub). While you're making this change, do you mind correcting it?

I wrongly thought that to add another key, you just add to the `sha256_cert_fingerprints` array for your package.

In fact, you should duplicate the entire object but with the new key, so the example below should be something like:

```
[{  
  "relation": ["delegate_permission/common.handle_all_urls"],  
  "target": {  
    "namespace": "android_app",  
    "package_name": "com.peter.trustedpetersactivity",
    "sha256_cert_fingerprints": [
      "FA:2A:03:CB:38:9C:F3:BE:28:E3:CA:7F:DA:2E:FA:4F:4A:96:F3:BC:45:2C:08:A2:16:A1:5D:FD:AB:46:BC:9D",
    ]
  }  
},{  
  "relation": ["delegate_permission/common.handle_all_urls"],  
  "target": {  
    "namespace": "android_app",  
    "package_name": "com.peter.trustedpetersactivity",
    "sha256_cert_fingerprints": [
      "4F:FF:49:FF:C6:1A:22:E3:BB:6F:E6:E1:E6:5B:40:17:55:C0:A9:F9:02:D9:BF:28:38:0B:AE:A7:46:A0:61:8C",
    ]
  }  
}]
```

Currently this works in Chrome because I implemented the check incorrectly. I'd like to change it, but unfortunately there are people relying on this behaviour. Once we have some channel to communicate with TWA developers (eg the Reporting API) we can warn them and make the change.

Reply:
Thanks for the info. Updated.